changelog/4358.breaking.rst

```diff
@@ -0,0 +1 @@
+:meth:`~sunpy.net.helio.HECClient.time_query` is removed; :meth:`~sunpy.net.helio.HECClient.search` is expected to be used instead.
```

Comment:
Is it possible to deprecate this rather than just remove it or is that too much work?

Reply:
Added it back.